docs/tutorial.rst

```diff
@@ -0,0 +1,1613 @@
+lower boundary for the glucose import, it will be defined in the limits file.
+An example can be seen below:
+
+.. code-block:: yaml
+
+    - reaction: R_ATPM
+      lower: 8.39
+
+Each entry in the limits file includes a reaction ID followed by upper and
+lower limits. Note that when a model is imported only the non default flux
+limits are explicitly stated, so some of the imported models will not contain
+a predefined limits file. In the example above, only one reaction has a non
+default limit. This reaction is an ATP maintenance and the modelers chose to
+force a certain level of flux through it to simulate the general energy cost of
+cellular maintenance.
+
+D) Medium
+~~~~~~~~~~~
+
+The medium file is where the user designates the boundary conditions for
+the model. A name for the media can be designated at the top of the file
+using the ‘name’ tag, and the compartment of the medium compounds can be
+designated using the ‘compartment’ tag. The media can be organized into
+multiple sections to represent exchange reactions for different compartments
+in the model if that is desired. An example of the medium file can be seen
+below.
+
+.. code-block:: yaml
+
+    name: Default medium
+    compartment: C_e
+    compounds:
+    - id: M_ac_e
+      reaction: R_EX_ac_e
+      lower: 0.0
+    - id: M_acald_e
+      reaction: R_EX_acald_e
+      lower: 0.0
+    - id: M_akg_e
+      reaction: R_EX_akg_e
+      lower: 0.0
+    - id: M_co2_e
+      reaction: R_EX_co2_e
+
+
+Each entry starts with the ID of the boundary compound and followed by lines
+that defines the lower and upper limits of the compound flux. Internally,
+`PSAMM` will translate these boundary compounds into exchange reactions in
+metabolic models. Additional properties can be designated for the exchange
+reactions including an ID for the reaction, the compartment for the reaction,
+and lower and upper flux bounds for the reaction. In the same way that only
+non standard limits need to be specified in the limits file, only non standard
+exchange limits need to be specified in the media file.
+
+
+E) Model Format Customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The YAML model format is highly customizable to suit the user's preferences.
+File names can be changed according to the user design. These customizations
+are all allowed by `PSAMM` as long as the central model.yaml file is also updated
+to reflect the different file names referred. While all the file names can be
+changed it is recommended that the central model.yaml file name does not change.
+`PSAMM` will automatically detect and read the information from the file if it is
+named model.yaml. If the user does wish to also alter this central file’s name
+they can do so but whenever any psamm commands are run they will need to specify
+the name of their model’s central file using the model option. For example run
+fba with a different central model file name “ecoli_model.yaml”, the user could
+run FBA in `PSAMM` like this:
```

Comment:
Perhaps here a more explicit explanation that this is an example and should not be entered into the terminal if you have not created the "ecoli_model.yaml" file.  Maybe change the wording of "For example run
+fba with a different central model file name “ecoli_model.yaml”, the user could
+run FBA in `PSAMM` like this:" to
 "If you created the central model file name "ecoli_model.yaml" the user could run FBA in `PSAMM` like this: